### PR TITLE
Fix golint using type in variable declaration

### PR DESCRIPTION
**Pull request by kkirsche**
This fixes:

    main_test.go:25:9: should omit type discoAliasFlag from declaration of var a; it will be inferred from the right-hand side

### CODE REVIEW

File: cmd/goacme/main_test.go [hunk 1]
```diff
@@ -22,15 +22,15 @@ func TestDiscoAliasFlag(t *testing.T) {
 		{defaultDisco, []string{"-d", "https://disco"}, "https://disco"},
 	}
 	for i, test := range tests {
-		var a discoAliasFlag = test.a
+		var daf = test.a
```
Comment (crhym3) [inline cmd/goacme/main_test.go]: do you mind keeping the same name, i.e. `a`?

Comment (kkirsche) [inline cmd/goacme/main_test.go]: I can, any reason though? Just want to understand the reasoning. Go from how I've understood things usually follows the principle of using the first letter from each major word in the represented type / variable. In this case `discoAliasFlag` becomes `daf`.

Comment (crhym3) [inline cmd/goacme/main_test.go]: it's not a rule set in stone. let me rephrase it. you're trying to fix a
golint suggestion. why make absolutely unrelated change, i.e. rename a
variable?

Comment (kkirsche) [inline cmd/goacme/main_test.go]: fair, I guess for me I read the use of the type there as a way to provide a description for what `a` actually means. By removing this it becomes less clear what this variable represents, which is why I moved it to `daf`

**Comment by kkirsche** [comment]
Updated. While I don't agree personally with it, as I feel it helps provide the context which the type had provided before, I respect your decision on that.

**Comment by crhym3** [comment]
Much better now, thanks. I just don't like mixing in commits which have no relation to what the pull request is about.

Squashing and merging.
